prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh

```diff
@@ -0,0 +1,248 @@
+	az account set --subscription "${AZURE_SUBSCRIPTION_ID}"
+}
+
+# Enable this while debugging to confirm The User's Desire To Provision
+# A New EventHub Namespace For Their Current Azure/K8S Context
+confirmConfiguration() {
+
+  # Log The Configuration Summary
+  shout "The following configuration will be used to provision the new EventHub Namespace - review for correctness before continuing!"
+  shout ""
+  shout "                              Azure Subscription    :  ${AZURE_SUBSCRIPTION}"
+  shout "                              Azure Resource Group  :  ${RS_GROUP}"
+  shout ""
+  shout "             New EventHub Namespace Name            :  ${EVENTHUB_NAMESPACE_NAME}"
+  shout "             New EventHub Namespace Location        :  ${EVENTHUB_NAMESPACE_LOCATION}"
+  shout "             New EventHub Namespace Throughput Min  :  ${EVENTHUB_NAMESPACE_MIN_THROUGHPUT_UNITS}"
+  shout "             New EventHub Namespace Throughput Max  :  ${EVENTHUB_NAMESPACE_MAX_THROUGHPUT_UNITS}"
+  shout "                       Kubernetes Secret Name       :  ${EVENTHUB_NAMESPACE_NAME}"
+  shout "                       Kubernetes Secret Namespace  :  ${K8S_SECRET_NAMESPACE}"
+  shout "            Kafka Channel Controller Pod Name       :  ${KAFKA_CHANNEL_CONTROLLER_NAME}"
+  shout "            Kafka Channel Controller Pod Namespace  :  ${KAFKA_CHANNEL_CONTROLLER_NAMESPACE}"
+  shout ""
+}
+
+
+# Create The EventHub Namespace Based On User's Current Azure / K8S Context
+createEventHubNamespace() {
+
+  # Execute The Azure EventHubs Namespace Creation Command & Handle The Results
+  shout "Creating New EventHubs Namespace... (takes several minutes - be patient :)"
+  if [[ $(cmdCreateEventHubNamespace) -eq 0 ]]; then
+    sleep 2
+    shout "Successfully Created New EventHub Namespace!"
+  else
+    shout "Failed To Create New EventHub Namespace - Exiting!"
+    exit 1
+  fi
+}
+
+
+# Load The EventHub Namespace's Authorization Key Information Into Global Variables
+loadAuthorizationKey() {
+
+   # Get The New EventHub Namespace's PrimaryConnectionString
+  shout "Loading The New EventHub Namespace's Authorization Key..."
+  local primaryConnectionString=""
+  primaryConnectionString=$(cmdNamespacePrimaryConnectionString)
+
+  # Populate The Kubernetes Secret Broker / Password Values
+  K8S_SECRET_BROKER=$(echo "${primaryConnectionString}" | sed -e "s/^Endpoint=.*sb:\/\/\(.*\)\/;.*$/\1:${KAFKA_BROKER_PORT}/")
+  K8S_SECRET_PASSWORD=${primaryConnectionString}
+}
+
+
+# Print The EventHub Namespace Secret (Contains The EventHub Namespace Auth Keys)
+createK8SSecretFile() {
+
+  shout "Creating a Kubernetes Secret override For The New EventHub Namespace..."
+  eventHubSecret=$(cmdCreateventHubNamespaceSecret)
+  echo "$eventHubSecret" > "${EVENTHUB_NAMESPACE_NAME}-secret.yaml"
+}
+
+#
+# Main Script Execution
+#
+
+# Authenticating in Azure
+azureAuthenticating
```

Comment:
`verifyPathDependencies` should come before `az` is used in `azureAuthenticating `

Reply:
Made this step the first one to get executed